janitor/timeseries.py

```diff
@@ -275,88 +294,72 @@ def flag_jumps(
     Create boolean column(s) that flag whether or not the change
     between consecutive rows exceeds a provided threshold.
 
-    Functional usage example:
-
-    ```python
-    import pandas as pd
-    import janitor.timeseries
-
-    df = pd.DataFrame(...)
-
-    df = flag_jumps(
-        df=df,
-        scale="absolute",
-        direction="any",
-        threshold=2,
-    )
-    ```
-
     Method chaining example:
 
     ```python
-    import pandas as pd
-    import janitor.timeseries
-
-    df = (
-        pd.DatFrame(...)
-        .flag_jumps(
-            scale="absolute",
-            direction="any",
-            threshold=2,
-        )
-    )
-    ```
-
-    Detailed chaining examples:
-
-    ```python
-    # Applies specified criteria across all columns of the DataFrame
-    # Appends a flag column for each column in the DataFrame
-    df = (
-        pd.DataFrame(...)
-        .flag_jumps(
-            scale="absolute",
-            direction="any",
-            threshold=2
-        )
-    )
+    >>> import pandas as pd
+    >>> import janitor.timeseries
+
+    >>> ts_index = pd.date_range("1/1/2019", periods=3, freq="1H")
+    >>> test_df = pd.DataFrame(
+    ...    {
+    ...         "col1": [*range(3)],
+    ...         "col2": [*range(0,6,2)],
+    ...         "col3": [20, 21, 42],
+    ...    },
+    ...    index=ts_index,
+    ... )
+    >>> test_df
+                         col1  col2  col3
+    2019-01-01 00:00:00     0     0    20
+    2019-01-01 01:00:00     1     2    21
+    2019-01-01 02:00:00     2     4    42
+    >>> test_df.flag_jumps(
+    ...        scale="absolute",
+    ...        direction="any",
+    ...        threshold=1.0,
+    ...    )
+                         col1  col2  col3  col1_jump_flag  col2_jump_flag  col3_jump_flag
+    2019-01-01 00:00:00     0     0    20               0               0               0
+    2019-01-01 01:00:00     1     2    21               0               1               0
+    2019-01-01 02:00:00     2     4    42               0               1               1
+    >>> test_df.flag_jumps(
+    ...     strict = True,
+    ...     scale=dict(col3="percentage"),
+    ...     direction=dict(col2="any"),
+    ...     threshold=dict(col2=.5),
+    ... )
+                         col1  col2  col3  col1_jump_flag  col2_jump_flag  col3_jump_flag
+    2019-01-01 00:00:00     0     0    20               0               0               0
+    2019-01-01 01:00:00     1     2    21               1               1               0
+    2019-01-01 02:00:00     2     4    42               1               1               1
 
     # Applies specific criteria to certain DataFrame columns
     # Applies default criteria to columns not specifically listed
     # Appends a flag column for each column in the DataFrame
```

Comment:
These don't render well currently in mkdocs (again, highly recommend you run `mkdocs` locally to see the output for yourself; make sure it looks good before pushing up your changes. :)

![image](https://user-images.githubusercontent.com/30731072/167280526-33bf3235-8577-4088-8337-a4a54c551805.png)